### DISCUSSION TITLE
Are our community chats good enough?

### DISCUSSION THREAD

**Discussion by MrNaif2018** [MrNaif2018/bitcart#286] [General]
Right now we have a telegram group and a discord server, bridged together.
As for the social media we have twitter and reddit

Is the current community experience good for you, or did anyone have any issues joining our community chats?

**Comment by trymeouteh** [comment]
Please add a matrix chat. Matrix is FOSS and federated and not centralised and proprietary like telegram and discord

**Reply by MrNaif2018** [reply]
I'll look into that. Do you think that matrix is better than a self-hosted mattermost instance?

**Reply by MrNaif2018** [reply]
@trymeouteh You can now join `@bitcartcc:matrix.org`
